apiserver/params/params.go

```diff
@@ -563,14 +564,15 @@ type UpdateApplicationUnits struct {
 
 // ApplicationUnitParams holds unit parameters used to update a unit.
 type ApplicationUnitParams struct {
-	ProviderId     string                     `json:"provider-id"`
-	UnitTag        string                     `json:"unit-tag"`
-	Address        string                     `json:"address"`
-	Ports          []string                   `json:"ports"`
-	FilesystemInfo []KubernetesFilesystemInfo `json:"filesystem-info,omitempty"`
-	Status         string                     `json:"status"`
-	Info           string                     `json:"info"`
-	Data           map[string]interface{}     `json:"data,omitempty"`
+	ProviderId      string                     `json:"provider-id"`
+	UnitTag         string                     `json:"unit-tag"`
+	Address         string                     `json:"address"`
+	Ports           []string                   `json:"ports"`
+	FilesystemInfo  []KubernetesFilesystemInfo `json:"filesystem-info,omitempty"`
+	Status          string                     `json:"status"`
+	Info            string                     `json:"info"`
+	Data            map[string]interface{}     `json:"data,omitempty"`
+	ContainerStatus status.StatusInfo          `json:"container-status"`
```

Comment:
params structs don't use non params things like status.StatusInfo
plus, the Status, Info, Data values are what will be container status